src/DependencyInjection/Configuration.php

```diff
@@ -156,8 +156,8 @@ public function getConfigTreeBuilder()
             ->fixXmlConfig('firewall_name')
             ->children()
                 ->arrayNode('firewall_names')
-                    ->isRequired()
-                    ->requiresAtLeastOneElement()
+                    ->setDeprecated('hwi/oauth-bundle', '2.0', '"%path%.%node%" is deprecated. Firewall names are collected automatically.')
```

Comment:
Well, as 2.0 is not yet final we could deprecate this in 1.x (1.5?) and remove it in 2.0

Reply:
I thought this commits will be for 2.0. is version 1.5.also planned?

Reply:
if required I can prepare commits for 1.x-dev too.

Reply:
I don't want to have any 1.5, I can leave with this deprecation in 2.0 :)